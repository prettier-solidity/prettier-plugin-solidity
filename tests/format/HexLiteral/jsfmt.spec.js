run_spec(__dirname, ['solidity-parse'], { singleQuote: true });
run_spec(__dirname, ['solidity-parse'], { singleQuote: false });
