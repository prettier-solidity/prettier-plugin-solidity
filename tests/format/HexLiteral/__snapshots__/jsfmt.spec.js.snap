// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HexLiteral.sol - {"compiler":"0.8.6","singleQuote":false} format 1`] = `
====================================options=====================================
compiler: "0.8.6"
parsers: ["solidity-parse"]
printWidth: 80
singleQuote: false
                                                                                | printWidth
=====================================input======================================
// SPDX-License-Identifier: MIT
pragma solidity 0.8.10;

contract HexLiteral {
  bytes8 hex1 = hex'DeadBeef';
  bytes8 hex2 = hex"DeadBeef";
}

=====================================output=====================================
// SPDX-License-Identifier: MIT
pragma solidity 0.8.10;

contract HexLiteral {
    bytes8 hex1 = hex"DeadBeef";
    bytes8 hex2 = hex"DeadBeef";
}

================================================================================
`;

exports[`HexLiteral.sol - {"compiler":"0.8.6","singleQuote":true} format 1`] = `
====================================options=====================================
compiler: "0.8.6"
parsers: ["solidity-parse"]
printWidth: 80
singleQuote: true
                                                                                | printWidth
=====================================input======================================
// SPDX-License-Identifier: MIT
pragma solidity 0.8.10;

contract HexLiteral {
  bytes8 hex1 = hex'DeadBeef';
  bytes8 hex2 = hex"DeadBeef";
}

=====================================output=====================================
// SPDX-License-Identifier: MIT
pragma solidity 0.8.10;

contract HexLiteral {
    bytes8 hex1 = hex'DeadBeef';
    bytes8 hex2 = hex'DeadBeef';
}

================================================================================
`;
